bitnami/matomo/values.yaml

```diff
@@ -688,6 +689,7 @@ externalDatabase:
   password: ""
   database: bitnami_matomo
   existingSecret: ""
+  existingSecretPasswordKey: db-password
```

Comment:
To maintain consistency with other assets, I think it's best for the existingSecretPasswordKey to be left empty. [Eg](https://github.com/bitnami/charts/blob/main/bitnami/jupyterhub/values.yaml#L1785)